### PR TITLE
Add OpenCollective + Smile donation buttons

### PR DESCRIPTION
**Pull request by wilzbach**
Preview:

image

### CODE REVIEW

**Comment by JackStouffer** [comment]
I know Mike has the CafePress, but do we have the t-shirts all set up?

**Comment by wilzbach** [comment]
> I know Mike has the CafePress, but do we have the t-shirts all set up?

Not entirely, but if worst comes to the worst I can order them in Germany. International shipping from Germany to anywhere in world is 10€.

File: foundation/donate.dd [hunk 1]
```diff
@@ -30,6 +30,18 @@ $(H3 How can I donate?)
 
 We $(LINK2 sponsors.html, welcome contributions) from both organizations and individuals. Your help is highly appreciated.
 
+$(DONATE_ITEM Donate through OpenCollective, users,
+
+    Make a $(I transparent) donation. Select the package you're comfortable with and get the T-shirt!
```
Comment (JackStouffer) [inline foundation/donate.dd]: This makes is sound like any donation gets a tshirt. I'd just leave this sentence out.

Comment (EmTee70) [reply foundation/donate.dd]: Yes!

Comment (wilzbach) [reply foundation/donate.dd]: Fixed.

**Comment by wilzbach** [comment]
Going to merge this now, s.t. it's visible to the general public. The design can always be improved in a follow-up.

**Pull request merged**